docroot/profiles/custom/cgov_site/themes/custom/ncids_trans/templates/cards/block-content--cgov-external-link-block--embedded-feature-card-no-image.html.twig

```diff
@@ -10,4 +10,10 @@
 {# ###### Setup URL ######## #}
 {% set url = content.field_featured_url[0]['#url'] %}
 
-{{ cardMacros.featureCard(title, description, null, url) }}
+{% set id = node.id()|clean_unique_id %}
```

Comment:
This is not correct. You are not a node entity, but you are a block_content entity. This should be throwing errors...